assets/src/styles/blocks/core-overrides/Table.scss

```diff
@@ -4,44 +4,59 @@
   table {
     // Grey background (defaut)
     th {
-      background-color: $table-header-grey;
+      background-color: var(--table-header-grey--background, $table-header-grey);
     }
 
     tr {
-      background-color: $table-even-row-grey;
+      background-color: var(--table-even-row-grey--background, $table-even-row-grey);
     }
 
     tfoot td {
-      background-color: $table-footer-grey;
+      background-color: var(--table-footer-grey--background, $table-footer-grey);
     }
 
     // Green background
     &.has-green-background-color {
       th {
-        background-color: $table-header-green;
+        background-color: var(--table-header-green--background, $table-header-green);
       }
 
       tr {
-        background-color: $table-even-row-green;
+        background-color: var(--table-even-row-green--background, $table-even-row-green);
       }
 
       tfoot td {
-        background-color: $table-footer-green;
+        background-color: var(--table-footer-green--background, $table-footer-green);
       }
     }
 
     // Blue background
     &.has-blue-background-color {
       th {
-        background-color: $table-header-blue;
+        background-color: var(--table-header-blue--background, $table-header-blue);
       }
 
       tr {
-        background-color: $table-even-row-blue;
+        background-color: var(--table-even-row-blue--background, $table-even-row-blue);
       }
 
       tfoot td {
-        background-color: $table-footer-blue;
+        background-color: var(--table-footer-blue--background, $table-footer-blue);
+      }
+    }
+
+    // GP-Green background
```

Comment:
Could you please add this new style in the `new-identity/style` file instead of here (like for the [Spreadsheet block](https://github.com/greenpeace/planet4-master-theme/blob/main/assets/src/scss/new-identity/style.scss#L57-L74))? It should only be added for the new identity, and these CSS variables won't be defined if the new identity styles are disabled so I'm not sure what the behaviour will be like

Reply:
😅 my bad, I forgot to move the code after testing. Thanks